### ISSUE TITLE
lighthause and <link rel="preload" as="script" href="......js">

### ISSUE THREAD

**Issue by GiambattistaBenacchio** [GoogleChrome/lighthouse#10359]
Lighthause continues to report file already deleted ftp, even after clean browser
(cokies, history, navigation).
The file (sniff2.js) was preloaded in the DOM (Head) with
<link rel = "preload" as = "script" href = "sniff2.js"> and then deleted by ftp. I suspect that this file persists somewhere (cache?) and normal browser cleaning does not delete it.
 I need help, run multiple attempts several times, rummaged in the cache, but I don't solve it.

page of site: https://gotilfondatore.it/003pagg/0pagg.html (being reviewed for transition to PWA)
System: Window 10
Chrome version 79.0.3945.130 (official build) (64-bit)
Version  Lighthouse 5.6.0 

I enclose to complete the deleted file (sniff2.js) which I replaced on the site with sniffg.js
//+++ range screen browser for img++++++++++++++++++++++++++++
var largbrowser = (window.innerWidth); 
var largescreen = (window.screen.width);
if (largbrowser < 320) {location.href="avviso1.html";}   //320 browser
if (largescreen < 320) {location.href="avviso0.html";}   //meno di 320 screen
//++++page read+++++++++++++++++++++++++++++++
if (typeof(localStorage.codice_Lettura) === "undefined"){localStorage.codice_Lettura = window.location.href;}
var Va = (localStorage.codice_Lettura);
ris = Va.split('/');
paglet = ris.pop();
numpaglet = parseInt(paglet);
numpaglet = Number(numpaglet);
//+++++share+++++++++++++++++++++++++++++++++++++++++++++++++++++
ulpfb="https://www.facebook.com/sharer/sharer.php?u=https://gotilfondatore.it/003pagg/"+numpaglet+"pagg.html";
ulptw="https://twitter.com/share?text=Ho+letto+questa+pagina+del+romanzo+Got+il+Fondatore:&url=https://gotilfondatore.it/003pagg/"+numpaglet+"pagg.html";

**Comment by patrickhulce** [comment]
Thanks for filing @GiambattistaBenacchio but this is unrelated to Lighthouse. Your server is still serving that file and referencing it in the HTML.

```
curl -vvvv https://gotilfondatore.it/003pagg/sniff2.js > /dev/null
> GET /003pagg/sniff2.js HTTP/1.1
> Host: gotilfondatore.it
> User-Agent: curl/7.64.1
> Accept: */*
>
< HTTP/1.1 200 OK
< Server: nginx
< Date: Thu, 20 Feb 2020 18:48:24 GMT
< Content-Type: application/javascript
< Content-Length: 1045
< Connection: keep-alive
< Last-Modified: Thu, 20 Feb 2020 06:32:14 GMT
< Accept-Ranges: bytes
< Content-Language: it
<
{ [1045 bytes data]
```

![image](https://user-images.githubusercontent.com/2301202/74967902-80af3580-53df-11ea-9806-c32f8f4c3850.png)


We can't really provide support for generic server updating issues. Best of luck to you figuring it out though!

**Comment by GiambattistaBenacchio** [comment]
Thanks, your answer has ruled out problems with client cache, on which I had focused my investigation, and I solved it!
The problem originated from an iframe of the html page, I did not quite understand the depth of the LH report, which evidently and logically does not limit itself to fathoming only the page relating to the report but also the source pages of the iframes
possibly present in the html page.
Thanks again.
